index.html

```diff
@@ -133,55 +133,74 @@ <h3>Terminology</h3>
+						</dt>
+						<dd>
+							<p>The Authored Publication Manifest is the serialization of the manifest that the author
+								provides with the digital publication (i.e., prior to be digital publication being
+								processed by a user agent). Note that the author does not have to be human; a machine
+								could automatically produce authored manifests for digital publications.</p>
+						</dd>
 
-				<p>It is possible that an authored manifest is the equivalent of the canonical manifest if there are no
-					ambiguities or missing information, but a canonical manifest only exists after a user agent has
-					inspected the authored manifest as part of the process of obtaining it.</p>
+						<dt>
+							<dfn data-lt="canonical manifest|canonical publication manifest">Canonical Manifest</dfn>
+						</dt>
+						<dd>
+							<p>The Canonical Publication Manifest is a version of the manifest created by user agents
+								when they <a href="#processing-manifest">process the authored manifest</a> and remove
+								all possible ambiguities and incorporate any missing values that can be inferred from
+								another source.</p>
+						</dd>
+					</dl>
+
+					<p>It is possible that an authored manifest is the equivalent of the canonical manifest if there are
+						no ambiguities or missing information, but a canonical manifest only exists after a user agent
+						has inspected the authored manifest as part of the process of obtaining it.</p>
+				</section>
 
-				<p>This part of the specification describes the requirements for creating an authored manifest,
-					regardless of the format of the digital publication. Rules for constructing a canonical manifest
-					from the authored manifest are defined for each specific implementation.</p>
+				<section id="manifest-jsonld">
+					<h4>JSON-LD Authoring and Processing</h4>
 
+					<p>The <a>authored manifest</a> enables information to be compactly expressed to simplify authoring.
+						It is not always required that object types be explicitly authored, for example, as these are
+						automatically generated during canonicalization when missing (see <a
+							href="#properties-value-categories"></a> for more information).</p>
+
+					<p>As a result, the information represented by the authored manifest might be less complete than the
+						one represented by the <a>canonical manifest</a>. If consistency between the two states is
+						desired, it is RECOMMENDED to avoid the compact forms in the authored manifest.</p>
+
+					<p>Similarly, it is OPTIONAL for a user agent to be a full JSON-LD processor. User agents only need
+						to be able to read the manifest and internalize the data following the processing steps defined
+						in this specification. As a result, use of [[!json-ld]] features not described in this
+						specification (e.g., <code>@base</code>, JSON literals) is not advised, as user agents might
+						ignore such features or process them incorrectly.</p>
```

Comment:
JSON-LD features "not described in this specification" won't be easy for implementors to pin down...

It's probably simpler (and more consistent with other JSON-LD-based specifications) to instead reference the JSON serialization/structure that must be used to be considered a valid "publication manifest document" (enforced by via the spec text and a non-normative JSON Schema perhaps).

[ActivityStreams](https://www.w3.org/TR/activitystreams-core/#syntaxconventions) is a great example of this approach:
> This specification describes a JSON-based [RFC7159] serialization syntax for the Activity Vocabulary that conforms to a subset of [JSON-LD] syntax constraints but does not require JSON-LD processing. While other serialization forms are possible, such alternatives are not discussed by this document.